.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,73 @@
+name: "\U0001F41E Bug report"
+description: Report an issue with Vite
+labels: [pending triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue, tell us in the description. Thanks!
+      placeholder: Bug description
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Describe the bug
```

Comment:
Should this be **Reproduction**, since we already have **Describe the bug** above?